libs/platform/src/lib/components/menu-button/menu-button.component.ts

```diff
@@ -0,0 +1,56 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+import { ButtonType } from '@fundamental-ngx/core';
+import { InputSize } from './../form/form-control';
+
+@Component({
+    selector: 'fdp-menu-button',
+    templateUrl: './menu-button.component.html',
+    styleUrls: ['./menu-button.component.scss'],
+})
+export class MenuButtonComponent {
```

Comment:
We have now base component that should have common properties already that this component can inherit so you would be able to remove Id (property start with lowercase btw.), displaySize, disabled